sklearn/tree/_export.py

```diff
@@ -943,6 +944,10 @@ def export_text(
         A list of length n_features containing the feature names.
         If None generic names will be used ("feature_0", "feature_1", ...).
 
+    class_names : list of arguments, default=None
```

Comment:
Could this be `list of str` instead of `list of arguments`? Like the parameter above, features names.

Reply:
I also wondered about this, but when you print the tree, it is converted directly to a string with the `_add_leaf()` function : https://github.com/scikit-learn/scikit-learn/blob/dc7ef61d0d58be84355b965481032d59f8f41e2e/sklearn/tree/_export.py#L1026-L1034

So I figured that any element in the list, of any type, is converted to a string.